src/humanpy/hum_kin1.py

```diff
@@ -0,0 +1,263 @@
+PACKAGE = 'humanpy'
+import numpy
+import rospy
+import rospkg
+import collections
+from scipy import signal
+from tf import transformations, LookupException, ConnectivityException, ExtrapolationException
+import humanpy
+
+BASE_FRAME = '/map'
+KIN_FRAME = '/head/skel_depth_frame'
```

Comment:
I'd prefer if you took these as constructor arguments, since this is specific to the TF configuration of our robot.